Formula/opencv.rb

```diff
@@ -59,14 +56,21 @@ def install
     # Reset PYTHONPATH, workaround for https://github.com/Homebrew/homebrew-science/pull/4885
     ENV.delete("PYTHONPATH")
 
+    # Remove bundled libraries to make sure formula dependencies are used
+    libdirs = %w[ffmpeg libjasper libjpeg libjpeg-turbo libpng libtiff libwebp openexr openjpeg protobuf tbb zlib]
+    libdirs.each { |l| (buildpath/"3rdparty"/l).rmtree }
```

Comment:
Remaining directories:
```
carotene cpufeatures ippicv ittnotify libtengine libtim-vx openvx quirc
include/opencl
include/vulkan
```

Some aren't used (like `cpufeatures` is Android-only) and Vulkan is disabled by default. Others aren't formulae like Intel libraries.